operator/cmd/operator/server.go

```diff
@@ -97,9 +99,10 @@ func run() {
 		log.Fatalf("Could not get apiserver config: %v", err)
 	}
 
+	namespaces := strings.Split(watchNS, ",")
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
-		Namespace:               watchNS,
+		NewCache:                cache.MultiNamespacedCacheBuilder(namespaces),
```

Comment:
the spec says if namespace is empty it will watch all namespaces.

Watching all namespaces is pretty desirable I think

Reply:
agreed. this would be really good to have.

Reply:
+1 IIRC that's the behaviour. We should document this as part of the CLI help text

Reply:
mark this as `do-not-merge`, as I found that I tried to with empty string slice for `watchedNamespace`, then I get error in operator logs:
```
error getting IstioOperator iop: unable to get: istio-system/example-istiocontrolplane because of unknown namespace for the cache
```